pyproject.toml

```diff
@@ -26,26 +26,24 @@ classifiers = [
     'Programming Language :: SQL',
     'Topic :: Office/Business :: Financial :: Accounting',
 ]
-packages = [
-    { include = "beangrow", from = "." },
+requires-python = '>= 3.10'
+dependencies = [
+    'numpy >= 1.26.4',
+    'pandas >= 2.2.2',
+    'matplotlib >= 3.9.0',
+    'beancount >= 2.3.6',
+    'scipy >= 1.13.1',
+    'beanprice >= 1.2.1',
+    'pandas-stubs >= 2.2.2.240514',
+    'matplotlib-stubs >= 0.2.0',
```

Comment:
`pandas-stubs` and `matplotlib-stubs` are required only for type checking, not at runtime, therefore it is not necessary for users to install them. I'll leave these out or I'll add them to a `dev` extra dependencies group. My preference would be to leave them out as other development dependencies are not listed.

Reply:
SGTM @dnicolodi I'll merge this quick and try it out though, can be done later

Reply:
You're right, I overlooked this when I converted it from poetry to PEP 621.
I've created a follow-up PR https://github.com/beancount/beangrow/pull/39 to fix this and also added `ruff` and `mypy` to the dev dependencies.